recipes/oyaml/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "oyaml" %}
+{% set version = "0.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5064e85aa38ec481b1c49e4941957c6c5e1961291ce276b064c5d6a85d833ec5
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+
+requirements:
+  host:
+    - pip
+    - python
+    - pyyaml
+  run:
+    - python
+    - pyyaml
+
+test:
+  imports:
+    - oyaml
+
+about:
+  home: https://github.com/wimglenn/oyaml
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
```

Comment:
The `license_file` isn't included in the source distribution, so can you please manually package it:

https://conda-forge.org/docs/meta.html#packaging-the-license-manually

and put the following link to the issue upstream to include the license in the source distribution as a comment in the recipe so that if the issue is fixed upstream, there is a reminder to change the recipe in the feedstock in future releases:

https://github.com/wimglenn/oyaml/issues/12